docker_scripts/pebble_cmd.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+# Generate entrypoint based on scenario.
+bash "/opt/ceph-container/bin/generate_entrypoint.sh"
+rm -f /opt/ceph-container/bin/generate_entrypoint.sh
+bash -n /opt/ceph-container/bin/*.sh
+
+# Execute Entrypoint script
+bash /opt/ceph-container/bin/entrypoint.sh
```

Comment:
So above we check *.sh via `bash -n` but here we execute only `entrypoint.sh` 
For clarity it would be great to have some doc comment on which scripts are being expected here